frontend/packages/dev-console/integration-tests/features/build-config/edit-build-config.feature

```diff
@@ -0,0 +1,94 @@
+              And user selects imagestream "<image_stream>" and tag "<tag>" in Build From section in Image Configuration
+              And user enters Name and Value as "<name>" and "<value>" respectively in Environment Variables
+              And user clicks Save button on Edit build Config page
+             Then user will see Git Repository as "<git_url>" and Build from as "<image_stream>:<tag>"
+
+        Examples:
+                  | git_url                                 | image_stream | tag | name | value |
+                  | https://github.com/sclorg/django-ex.git | python       | 3.8 | path | /home |
+
+
+        @regression @manual
+        Scenario: Edit Advanced git options of buildconfig which uses strategy Docker and Git as source: EBC-01-TC05
+        # user can use buildconfig-with-strategy-docker-source-git.yaml from testData/yamls/BuildConfig
+            Given user has applied the YAML buildconfig-with-strategy-docker-source-git.yaml
+              And user is at Edit Build Config page
+             When user clicks Show advanced Git options
+              And user changes value of Context Dir to "beginner/static-site"
+              And user clicks Save button on Edit Build Config page
+             Then user will see Context dir as "beginner/static-site"
+
+
+        @regression @manual
+        Scenario Outline: Edit a buildconfig which uses strategy Docker and Dockerfile as source: EBC-01-TC06
+        # user can use buildconfig-with-strategy-docker-source-dockerfile.yaml from testData/yamls/BuildConfig
+            Given user has applied the YAML buildconfig-with-strategy-docker-source-dockerfile.yaml
+             When user clicks on action menu of build config
+              And user selects the option Edit BuildConfig
+              And user enters Name and Value as "<name>" and "<value>" respectively in Environment Variables
+              And user clicks Save button on Edit build Config page
+              And user goes to Environment tab
+             Then user will see Name as "<name>" and and Value as "<value>"
+
+        Examples:
+                  | name | value |
+                  | path | /home |
+
+
+        @regression @manual
+        Scenario Outline: Edit a buildconfig which uses strategy Source and Binary as source: EBC-01-TC07
+        # user can use buildconfig-with-strategy-source-source-binary.yaml from testData/yamls/BuildConfig
+            Given user has applied the YAML buildconfig-with-strategy-source-source-binary.yaml
+             When user clicks on action menu of build config
+              And user selects the option Edit BuildConfig
+              And user enters Name and Value as "<name>" and "<value>" respectively in Environment Variables
+              And user clicks Save button on Edit build Config page
+              And user goes to Environment tab
+             Then user will see Name as "<name>" and and Value as "<value>"
```

Comment:
Can't we automate this? It seems to me that we can automate all the statements, the first one as well.

Reply:
I thought we are not doing this in Add flows so marked as manual. Updated